registry/config/default.ts

```diff
@@ -4,7 +4,7 @@ module.exports = {
         client: 'sqlite3',
         connection: {
             database: 'registry_db',
-            filename: './server/mydb.sqlite',
+            filename: './server/dbfiles/db.sqlite',
```

Comment:
Did you move this sqlite file to that folder?

Reply:
Yes, we need this to simplify integration with Docker. It has some issues with file mounting while for folders everything works fine. So instead of mounting `./server/mydb.sqlite` now can mount `./server/dbfiles/` folder.